PowerPointLabs/PowerPointLabs/PasteLab/PasteLabMain.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Windows;
+
+using PowerPointLabs.Utils;
+
+using PowerPoint = Microsoft.Office.Interop.PowerPoint;
+
+namespace PowerPointLabs.PasteLab
+{
+    public class PasteLabMain
```

Comment:
some principles i follow:
logs events in a way that, when u read the log file
1. you can understand the steps to reproduce what a user has done
2. when ur feature contains states, better record them in the lifecycle (begin, end, any state update, encounter error etc) so that to also reproduce the states
3. you know what's the error and exception (error msg, stacktrace etc) being thrown